client/src/Pos/Client/KeyStorage.hs

```diff
@@ -3,117 +3,138 @@
 {-# LANGUAGE TypeFamilies #-}
 
 module Pos.Client.KeyStorage
-       ( MonadKeysRead (..)
-       , MonadKeys (..)
-       , getSecretDefault
-       , modifySecretPureDefault
-       , modifySecretDefault
-
-       , getPrimaryKey
-       , getSecretKeys
-       , getSecretKeysPlain
-       , addSecretKey
-       , deleteAllSecretKeys
-       , deleteSecretKeyBy
-       , newSecretKey
-       , KeyData
-       , KeyError (..)
-       , AllUserSecrets (..)
-       , keyDataFromFile
-       ) where
+    ( MonadKeysRead (..)
```

Comment:
Does this whitespace change actually contain anything other that just whitespace changes? Its hard to tell.

On top of that, the new version does not conform to layout used in the rest of the code base. Please correct this.